src/api/libcellml/operatorlibrary.h

```diff
@@ -0,0 +1,156 @@
+#include <iomanip>
+#include <memory>
+#include <sstream>
+#include <string>
+
+namespace libcellml {
+
+namespace operators {
+
+class Representable
```

Comment:
Great work! Really happy to see this added. Will need lots of documentation though! (Not everything makes immediate sense to me without it: What's a Representable? What's a STDOperator?)

Reply:
Thank you :) The documentation will be added soon.
`Representable` is an umbrella class for anything that can be represented by the code generator. Think of it as a "Generatable" class. On C++ side, it's anything that has a `repr` method, which is the code representation of an object.
`STDOperator` is anything whose representation is in the form `std::xxx`, so anything that is provided by the C++ standard library.

Reply:
We definitely need to add the documentation to these classes.

Reply:
Some documentation is now in place.